tools/src/main/scala/scala/scalanative/linker/Reach.scala

```diff
@@ -99,6 +102,28 @@ class Reach(config: build.Config, entries: Seq[Global], loader: ClassLoader) {
       }
     }
 
+  def processDelayed(): Unit = {
+    /*  Check methods that were marked to not have any defined targets yet when processing loop.
+     *  At this stage they should define at least 1 target, or should be marked as a missing symbol.
+     */
+    delayedMethods.foreach {
+      case DelayedMethod(top, sig, position) =>
+        scopeInfo(top).foreach { info =>
+          val wasAllocated = info match {
+            case value: Trait => value.implementors.exists(_.allocated)
+            case clazz: Class => clazz.allocated
+          }
+          val targets = info.targets(sig)
+          if (targets.isEmpty && wasAllocated) {
+            addMissing(top.member(sig), position)
+          } else {
+            todo ++= targets
+          }
+        }
+    }
+    process()
```

Comment:
That second `process()` could cause new stuff to be reached, hence added to `delayedMethods`, and those would still not be treated.

It looks like we need either
* a loop, doing `process()` and `processDelayed()` until there is nothing left, or
* when finding a new method, immediately check whether it is in the `delayedMethods` set, and extract it to put in the todo